CONTRIBUTING.md

```diff
@@ -2,52 +2,75 @@
 
 ## Code of Conduct
 
-All interactions with this project follow our [Code of Conduct][code-of-conduct].
-By participating, you are expected to honor this code. Violators can be banned
-from further participation in this project, or potentially all Coinbase projects.
+All interactions with this project follow our [Code of Conduct][code-of-conduct]. By participating, you are expected to honor this code. Violators can be banned from further participation in this project, or potentially all Coinbase projects.
 
 [code-of-conduct]: https://github.com/coinbase/code-of-conduct
 
 ## Bug Reports
 
-* Ensure your issue [has not already been reported][1]. It may already be fixed!
-* Include the steps you carried out to produce the problem.
-* Include the behavior you observed along with the behavior you expected, and
-  why you expected it.
-* Include any relevant stack traces or debugging output.
+Before filing a bug report, ensure that your issue [has not already been reported][1]. It may already be fixed!
+
+If your bug hasn’t been fixed, follow these steps to file a bug report:
+
+1. If your issue is not reported or fixed, click the [*New*][5] button from the [*Issues*][1] page. A new interface will appear.
```

Comment:
`A new interface will appear`  seems unnecessary to me on both point 1 and 2. Github is quite intuitive and generic in that sense that we don't need to tell that.